lib/ukdebug/Makefile.uk

```diff
@@ -16,6 +16,8 @@ LIBUKDEBUG_SRCS-y += $(LIBUKDEBUG_BASE)/hexdump.c
 LIBUKDEBUG_SRCS-$(CONFIG_LIBZYDIS) += $(LIBUKDEBUG_BASE)/asmdump.c
 
 LIBUKDEBUG_SRCS-$(CONFIG_LIBUKDEBUG_GDBSTUB) += $(LIBUKDEBUG_BASE)/gdbstub.c|isr
+LIBUKDEBUG_SRCS-$(CONFIG_LIBUKDEBUG_GDBSTUB) += $(LIBUKDEBUG_BASE)/gdbtargetxml.ld
+LIBUKDEBUG_SRCS-$(CONFIG_LIBUKDEBUG_GDBSTUB) += $(LIBUKDEBUG_BASE)/gdbtargetxml.S
```

Comment:
Can you add the following line:
```
LIBUKDEBUG_GDBTARGETXML_CDEPS += $(LIBUKDEBUG_BASE)/arch/$(CONFIG_UK_ARCH)/target.xml
```
This makes sure that any modifications to the xml file will retrigger a compilation of the assembler file that embeds the xml file.